agent/cleanup.sh

```diff
@@ -15,17 +15,16 @@ early_cleanup_validation
 rm -rf "${OCP_DIR}/manifests"
 rm -rf "${OCP_DIR}/output"
 
-case "${AGENT_E2E_TEST_BOOT_MODE}" in
-  "PXE" )
-    sudo pkill agentpxeserver || true
-    rm -rf ${WORKING_DIR}/boot-artifacts
-    ;;
-  "DISKIMAGE" )
+if [[ "${AGENT_E2E_TEST_BOOT_MODE}" == "DISKIMAGE" ]]; then
     sudo rm -rf "${OCP_DIR}/cache"
     sudo rm -rf "${OCP_DIR}/temp"
     sudo podman rmi -f ${APPLIANCE_IMAGE} || true
-    ;;
-esac
+fi
+
+if [[ -d ${BOOT_SERVER_DIR} ]]; then
```

Comment:
Isn't equivalent to the previous case (which I found more readable)? Ie, the pxe boot server gets currently created only when AGENT_E2E_TEST_BOOT_MODE is PXE, isn't it?

Reply:
No we also need to create the boot server for minimal ISO in the disconnected case. Its similar to PXE in that a http server is providing the files written to `boot-artifacts`, but its not the same set of files for minimal ISO